docs/synapse/userguides/ug012_storm_ref_datamod.rst

```diff
@@ -256,7 +256,52 @@ None.
 delprop()
 ---------
 
-Todo
+Delete a property from the specified node(s).
+
+**Operator Syntax:**
+
+.. parsed-literal::
+  *<query>* **delprop(** *<prop>*, [ *force=<n>* ] **)**
+
+**Macro Syntax:**
+
+.. parsed-literal::
+  *<query>* **[ -:** *<prop>* ... **]**
+
+**Examples:**
+::
+  # Operator examples
+
+  inet:fqdn = vertex.link delprop(:created)
+
+  inet:fqdn = vertex.link delprop(:created, force=1)
+
+  # Macro examples
+
+  inet:fqdn = vertex.link [ -:created ]
+
+  inet:fqdn = vertex.link [ -:created -:updated ]
+
+**Usage Notes:**
+
+* ``delprop()`` operates on the output of a previous query.  It does not consume the tufos from the query.
```

Comment:
While "tufo" is a correct technical term from a dev standpoint, we decided for user docs to use the terms "nodes" and "forms" which would be more familiar to non-devs. Suggest "It does not consume the nodes from the query..." instead of "tufos". 

That said, not entirely clear on "it does not consume the nodes"...if it takes them as input, it "consumes" them, yes? It just doesn't operate on them unless you specify force=1 ?

Reply:
re: consumption - some operators consume nodes by removing them from the current set of nodes in the quer and then doing processing with them.  Sometimes these actions may readd the nodes, or add an entirely different set of nodes all together.  We should have a clear way to say "This operator eats all the input nodes" vs "this operator leaves the input nodes intact".